src/material-experimental/mdc-radio/radio.scss

```diff
@@ -9,21 +9,28 @@
 @include mdc-radio.without-ripple($query: mdc-helpers.$mat-base-styles-without-animation-query);
 @include mdc-form-field.core-styles($query: mdc-helpers.$mat-base-styles-query);
 
-// This is necessary because we do not depend on MDC's ripple, but have our own that should be
-// positioned correctly. This can be removed once we start using MDC's ripple implementation.
-.mat-mdc-radio-button .mat-radio-ripple {
-  @include layout-common.fill;
-
-  pointer-events: none;
-  border-radius: 50%;
+.mat-mdc-radio-button {
+  &:not(._mat-animation-noopable) {
+    @include mdc-radio.without-ripple($query: animation);
+  }
 
-  .mat-ripple-element:not(.mat-radio-persistent-ripple) {
-    opacity: mdc-radio-theme.$ripple-opacity;
+  .mdc-radio {
+    // MDC theme styles also include structural styles so we have to include the theme at least
+    // once here. The values will be overwritten by our own theme file afterwards.
+    @include mdc-radio-theme.theme-styles(mdc-radio-theme.$light-theme);
```

Comment:
Should we turn off fallback values going forward, and just tell the theming system to slot the variables?

Reply:
This is the mixin that slots the variables. I'd rather keep the fallbacks for now and we can have a separate PR that turns them off for all the components using the new API.